### ISSUE TITLE
Update Catalog links

### ISSUE THREAD

**Issue by lizzycederberg** [usnationalarchives/nara-bia-photos#251]
National Archives ID link should open in a new tab (expected from the icon used). Also, there should be some textual/screen reader information available that this will take the user to catalog.archives.gov. Same needs to be applied to “view all in the catalog”.
